src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/services/GetRiskPredictorScoresForPersonService.kt

```diff
@@ -19,6 +20,9 @@ class GetRiskPredictorScoresForPersonService(
 
     if (deliusCrn != null) {
       personRiskPredictorScores = assessRisksAndNeedsGateway.getRiskPredictorScoresForPerson(id = deliusCrn)
+      if (personRiskPredictorScores.hasError(UpstreamApiError.Type.FORBIDDEN)) {
```

Comment:
This if statement is not required as it will return the same result as per line 24 when personRiskPredictorScores has error